manager.py

```diff
@@ -335,15 +335,20 @@ def genweight_by_nick(self, nick):
         sample = self.database[nick]
         questionary.print(f"--- {nick} ---", style="bold")
         questionary.print(f"Current generator_weight: {sample['generator_weight']}")
-        questionary.print(f"Will calcuate new generator_weight for the sameple (this will take some minutes ....)")
+        questionary.print(
+            f"Will calcuate new generator_weight for the sameple (this will take some minutes ....)"
```

Comment:
Fixing typo in printout.
```suggestion
            f"Will calculate new generator_weight for the sample (this will take some minutes ....)"
```

Reply:
How?

Reply:
chill, commit will come in a sec